src/Database/LSMTree/Internal/RunAcc.hs

```diff
@@ -298,6 +298,12 @@ flushPageIfNonEmpty RunAcc{mpageacc, mindex} = do
 
       else pure Nothing
 
+{-# NOINLINE bloomInserts #-}
```

Comment:
Why `NOINLINE`?

Reply:
Initially it was just so I could see the generated core with a known name (rather than inlined at the use site). It doesn't otherwise hurt or benefit.